frontend/components/PrintAreaCommand/_index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="PrintAreaCommand">
```

Comment:
Since printing is an action, should it ever be a link?

Reply:
@kelliewellingtons suggested that we should only have one primary action on the page which will be to print and we could offer the printing of a particular section as links.